remotespark/kernels/kernelmagics.py

```diff
@@ -127,6 +152,7 @@ def logs(self, line, cell="", local_ns=None):
     @magic_arguments()
     @cell_magic
     @argument("-f", "--force", type=bool, default=False, nargs="?", const=True, help="If present, user understands.")
+    @_event
     def configure(self, line, cell="", local_ns=None):
         args = parse_argstring(self.configure, line)
         if self.session_started:
```

Comment:
For configure, is it useful to make the event tell us if a session was restarted?